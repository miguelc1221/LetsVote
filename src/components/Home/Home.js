import React, { Component } from 'react';
import { Header, Button } from 'semantic-ui-react'

import './Home.css';

class Home extends Component {
  render() {
    return (
      <div className='home'>
        <div className='home__jumbo'>
          <Header textAlign='center' className='home__jumbo-header'>
            <Header.Content as='h1'>
              Let's Vote
            </Header.Content>
            <Header.Subheader as='h2'>
              Create Custom Polls And Share Them With Friends
            </Header.Subheader>
            <Header>
              <Button color='olive'>Sign up</Button>
            </Header>
          </Header>
        </div>
        <div className='home__get-started'>
          <div className='home__get-started-innerContainer'>
            <h1>Getting started</h1>
            <div className='unordered-list'>
              <ol>
                <li>Sign up</li>
                <li>Create a new poll</li>
                <li>Share given link with participants</li>
                <li>Check results under (My Polls) </li>
              </ol>
            </div>
          </div>
        </div>
      </div>
    );
  }
}

export default Home;