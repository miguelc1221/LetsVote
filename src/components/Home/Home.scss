.home {
  display: flex;
  flex-direction: column;
  flex: 1;
  &__jumbo {
    text-align: center;
    min-height: 450px;
    background-image: url('../../../public/images/hands.png');
    background-repeat: no-repeat;
    background-size: contain;
    background-position: bottom;
  }
  &__jumbo-header {
    padding-top: 120px !important;
    h1 {
      font-size: 45px;
    }
    h2 {
      font-size: 20px !important;
    }
  }
  &__get-started {
    background-color: #F5F8FA;
    margin-top: -40px;
    padding: 35px;
    flex: 1;
  }
  &__get-started-innerContainer {
    width: 250px;
    margin: 0 auto;
    h1 {
      text-align: center;
    }
    div ol {
      line-height: 2;
    }
  }
}