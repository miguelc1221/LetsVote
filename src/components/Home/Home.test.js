import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import { Home } from './Home';

describe('Home Component', () => {
  it('should render as expected', () => {
    const component = shallow(<Home />);
    const tree = toJson(component);
    expect(tree).toMatchSnapshot();
  })
})