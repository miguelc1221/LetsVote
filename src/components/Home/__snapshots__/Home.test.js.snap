exports[`Home Component should render as expected 1`] = `
<div
  className="home">
  <div
    className="home__jumbo">
    <Header
      className="home__jumbo-header"
      textAlign="center">
      <HeaderContent
        as="h1">
        Let\'s Vote
      </HeaderContent>
      <HeaderSubheader
        as="h2">
        Create Custom Polls And Share Them With Friends
      </HeaderSubheader>
      <Header>
        <Connect(Signup) />
      </Header>
    </Header>
  </div>
  <div
    className="home__get-started">
    <div
      className="home__get-started-innerContainer">
      <h1>
        Getting started
      </h1>
      <div
        className="unordered-list">
        <ol>
          <li>
            Sign up
          </li>
          <li>
            Create a new poll
          </li>
          <li>
            Share given link with participants
          </li>
          <li>
            Check results under (My Polls) 
          </li>
        </ol>
      </div>
    </div>
  </div>
</div>
`;
